proto/network_state.proto

```diff
@@ -8,6 +8,7 @@ message NetworkState {
 message DeviceLearning {
   string switch = 1;
   int32 port = 2;
+  bool connecting = 3;
```

Comment:
connected not connecting -- state is absolute.

Reply:
Done